.github/workflows/ci.yml

```diff
@@ -1,6 +1,10 @@
 name: Continuous integration
 on: [push, pull_request]
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
```

Comment:
Why this? Will this stop all workflows if one fails? If so, I would rather not have it as it's useful to know e.g. about clippy issues even if tests fail.

Reply:
I see, then it makes sense.